test/unit_tests/arkode/CXX_serial/ark_test_dahlquist_ark.cpp

```diff
@@ -363,13 +395,13 @@ int main(int argc, char* argv[])
      "ARKODE_ARK324L2SA_DIRK_4_2_3",
      "ARKODE_ARK436L2SA_DIRK_6_3_4",
      "ARKODE_ARK437L2SA_DIRK_7_3_4",
-     "ARKODE_ARK548L2SA_ERK_8_4_5",
-     "ARKODE_ARK548L2SAb_ERK_8_4_5"};
+     "ARKODE_ARK548L2SA_DIRK_8_4_5",
+     "ARKODE_ARK548L2SAb_DIRK_8_4_5"};
 
   for (int i = 0; i < 6; i++)
   {
-    Be = ARKodeButcherTable_LoadERKByName(ark_methods_erk[0]);
-    Bi = ARKodeButcherTable_LoadDIRKByName(ark_methods_dirk[0]);
+    Be = ARKodeButcherTable_LoadERKByName(ark_methods_erk[i]);
+    Bi = ARKodeButcherTable_LoadDIRKByName(ark_methods_dirk[i]);
```

Comment:
This is the actual bugfix. The other changes are to make some variable/constant names clearer and add an option to change the predictor method for future testing.